src/endpoint/s3/s3_bucket_logging.js

```diff
@@ -34,6 +36,18 @@ function endpoint_bucket_op_logs(op_name, req, res, source_bucket) {
     const s3_log = get_bucket_log_record(op_name, source_bucket, req, res);
     dbg.log1("Bucket operation logs = ", s3_log);
 
+    const buffer = Buffer.from(JSON.stringify(s3_log));
+    const client = dgram.createSocket('udp4');
```

Comment:
Can we have a single udp socket object instead of re-creating it for each request?

Reply:
done